Mod Menu Helper/assets/fo/lang/zh_cn.json

```
@@ -0,0 +1,57 @@
+{
+  "modmenu.dropInfo.line1": "FO模组有§9✎§r或§8✎§r，代替品有§c*§r;",
+  "modmenu.dropInfo.line2": "其他不受支持",
+  "modmenu.summaryTranslation.advancementinfo": "更大、更智能的进度屏幕\n§9✎§8颜色、信息框大小……",
+  "modmenu.summaryTranslation.advancements-enlarger": "§c*§r更大的进步屏幕\n§8✎（没有选项）",
+  "modmenu.summaryTranslation.animatica": "OptiFine的动画纹理\n§8✎（只有高级选项）",
+  "modmenu.summaryTranslation.antighost": "使用/ghost\n§9✎§8键绑定修复幽灵方块",
```

Lines 7 and 8 should be partly identical
```suggestion
  "modmenu.summaryTranslation.antighost": "使用/ghost修复幽灵方块\n§9✎§8键绑定修复幽灵方块",
```